Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1092,6 +1118,15 @@ def get_report_summary_command(
 
     for single_id in argToList(ids):
         response = client.get_report_summary(single_id)
+        if response.get('resources'):
+            is_polling = argToBoolean(polling)
+            # We can extract the error from the response object under resources
+            error_message = validate_sandbox_report(response.get('resources', []))
+            if error_message:
+                if is_polling:
+                    raise DemistoException(error_message)
+                if not is_polling:
+                    return_warning(error_message)
```

Comment:
The condition checking if we are using polling may be redundant, since we use the command `cs-fx-get-full-report` to retrieve the report of the submitted file when polling, and not the `cs-fx-get-report-summary` command (the function `get_report_summary_command` is the entry point for the `cs-fx-get-report-summary` command)